homeassistant/components/prometheus/__init__.py

```diff
@@ -150,9 +154,10 @@ def handle_event(self, event):
         )
         metric.labels(**self._labels(state)).inc()
 
-    def _metric(self, metric, factory, documentation, labels=None):
-        if labels is None:
-            labels = ["entity", "friendly_name", "domain"]
+    def _metric(self, metric, factory, documentation, extra_labels=None):
+        labels = ["entity", "friendly_name", "domain"]
```

Comment:
This seems like it's changing logic unrelated to this PR.

Reply:
I don't believe this function ever gets called with `extra_labels` set.

Reply:
They're passing `action` as an extra label, so it is related.

Reply:
It's removing the posibility to overwrite all labels, though I'm not sure that is being used anywhere.

Reply:
As I mentioned above, that argument was never used. I have checked that.